python/servo/command_base.py

```diff
@@ -205,6 +205,21 @@ def is_macosx():
     return sys.platform == 'darwin'
 
 
+def find_native_win32_python():
+    paths = subprocess.Popen(['where', 'python.exe'],
```

Comment:
I'm pretty sure `which` is present in most of the distributions. I'm curious whether we can rely on `where`